frontend/src/views/Admin/AdminMenu/components/AuditLog.jsx

```diff
@@ -33,12 +35,26 @@ export const AuditLog = () => {
     []
   )
 
+  const handleSetResetGrid = useCallback((fn) => {
+    setResetGridFn(() => fn)
+  }, [])
```

Comment:
Do you need the most inner function? I think this can just be setResetGridFn(fn)

Reply:
Yes, I need the inner function, it allows to have an stable function reference:
![image](https://github.com/user-attachments/assets/874eeca8-59b0-486e-94a9-9ca5b0e91fda)

If I update it to just include `fn` instead of the arrow function, then I am loosing the function reference:
![image](https://github.com/user-attachments/assets/0145f598-8d0b-4bf6-944b-77d8934e566d)

When clicking the button, the function is not executed.